src/richie-front/js/utils/searchSuggest/suggestionHumanName.ts

```diff
@@ -2,17 +2,5 @@ import { modelName } from '../../types/models';
 import { ResourceSuggestion } from '../../types/searchSuggest';
 
 export const suggestionHumanName = (suggestion: ResourceSuggestion) => {
```

Comment:
I was going to suggest we move this to the call site and drop the tests...

... but then I though we might add People to the suggestions and their human name will probably not be `title`.